Source/Src_nd/diag.F90

```diff
@@ -79,66 +82,102 @@ subroutine diag(lev, lo, hi, &
 
     type (eos_t) :: eos_state
 
-
     ! weight is the factor by which the volume of a cell at the current level
     ! relates to the volume of a cell at the coarsest level of refinement.
-#if (AMREX_SPACEDIM == 2)
+#if (AMREX_SPACEDIM == 1)
```

Comment:
See previous comment: we've dropped 1d support so this needs to be removed

Reply:
Done